src/main/webapp/resources/js/contexts/AnalysisContext.js

```diff
@@ -0,0 +1,69 @@
+/*
+ * This file loads basic analysis info from the server.
+ */
+
+import React, { useState } from "react";
+import { showNotification } from "../modules/notifications";
+
+// Functions required by context
+import { updateAnalysis } from "../apis/analysis/analysis";
+
+/*
+ * Since we are using Steps and only want to display
+ * certain ones, we group some of the analysis states
+ * together.
+ */
+export const stateMap = {
+  NEW: 0,
+  PREPARING: 1,
+  SUBMITTING: 2,
+  RUNNING: 3,
+  FINISHED_RUNNING: 3,
+  POST_PROCESSING: 4,
+  TRANSFERRING: 4,
+  COMPLETING: 4,
+  COMPLETED: 5
+};
+
+export const isAdmin = window.PAGE.isAdmin;
+
+const initialContext = {
+  analysis: window.PAGE.analysis,
+  analysisName: window.PAGE.analysisName,
+  analysisState: window.PAGE.analysisState,
+  analysisType: window.PAGE.analysisType,
+  isCompleted: window.PAGE.analysisState == "COMPLETED",
```

Comment:
`===`

Reply:
Fixed in 2804c23baa10fdf170ebd8b07a191142d0927b12